server/services/prisonerProfileService.test.ts

```diff
@@ -63,26 +67,24 @@ describe('Prisoner profile service', () => {
     })
 
     it('Retrieves and processes data for prisoner profile with visit balances', async () => {
-      const bookings = <PagePrisonerBookingSummary>{
-        content: [TestData.prisonerBookingSummary()],
-        numberOfElements: 1,
-      }
-
-      const inmateDetail = TestData.inmateDetail()
-      const prisoner = TestData.prisoner()
-
-      const visitBalances: VisitBalances = {
-        remainingVo: 1,
-        remainingPvo: 2,
-        latestIepAdjustDate: '2021-04-21',
-        latestPrivIepAdjustDate: '2021-12-01',
-      }
-
-      const pagedVisit: PageVisitDto = {
-        totalPages: 1,
-        totalElements: 1,
-        size: 1,
-        content: [
+      const fullPrisoner = <PrisonerProfile>{
```

Comment:
We should get a `PrisonerProfile` into the `testData` file for re-use and consistency but that can be done as a subsequent change